charts/kube-plex/templates/deployment.yaml

```diff
@@ -133,6 +133,32 @@ spec:
         - name: "NO_PROXY"
           value: "{{.Values.proxy.noproxy}}"
   {{- end }}
+{{- end }}
+# Official kube-plex container environment variables
+{{- if .Values.advertiseIp }}
+        - name: "ADVERTISE_IP"
+          value: "{{.Values.advertiseIp}}"
```

Comment:
These vars seem unrelated to the `PLEX_PREFERENCES` bit right?

I'm a bit conscious that PLEX_PREFERENCES doesn't really 'protect' users from misconfigurations in any way, and could potentially lead to someone doing something 'bad' (e.g. setting ALLOWED_NETWORKS here, as well as setting PLEX_PREFERENCES_ALLOWED_NETWORKS).

Would it make more sense for us to remove `PLEX_PREFERENCES_` and instead just add fields here for each preference we want to allow users to modify? If we have a standard pattern for how this is done, it should be trivial for us to extend the options we support in future